packages/wix-ui-tpa/package.json

```
@@ -67,7 +64,12 @@
     "-----release": "",
     "storybook:release": "npx janet deploy --ci || exit 0",
     "release-candidate": "node scripts/rc.js",
-    "changelog": "conventional-changelog --preset angular -l wix-ui-tpa -t WUT-v -i CHANGELOG.md -s --commit-path=."
+    "changelog": "conventional-changelog --preset angular -l wix-ui-tpa -t WUT-v -i CHANGELOG.md -s --commit-path=.",
+    "-----publish": "",
+    "postpublish": "npx npm-run-all -c -p --aggregate-output postpublish:storybook postpublish:tag",
```

I'd not add `npx npm-run-all`
it will take time to install this dependency and that time is likely gonna be longer than just running tasks in sequence. These tasks are not taking a long time.

also i'd suggest to use full name of CLI flags, so instead of `-c` just use `--continue-on-error`. longer name but it saves time whenever i will inevitably forget what is `-c` and will have to look it up

but really, imo this is more than enough:
```suggestion
    "postpublish": "npm run postpublish:storybook | npm run postpublish:tag",
```

---

Do you mean 
```
"postpublish": "npm run postpublish:storybook & npm run postpublish:tag",
```
?

---

or 
```
"postpublish": "npm run postpublish:storybook & npm run postpublish:tag",
```

---

But also, in `postpublish:storybook` we run npm install, and in `postpublish:tag` we run `git clone`.
These are actually pretty heavy scripts, so I think using `npx npm-run-all` will be a bit safer.

---

sorry, i mean with `&` not with `|`

i see you changed it, so it's ok we can use `npm-run-all`. but if it were me, i wouldn't add it until (if ever) there's a problem

---

thanks!